packages/kit/src/core/generate_manifest/index.js

```diff
@@ -65,8 +65,8 @@ export function generate_manifest({ build_data, relative_path, routes }) {
 
 	// prettier-ignore
 	// String representation of
-	/** @type {import('@sveltejs/kit').SSRManifest} */
-	return dedent`
+	/** @template {import('@sveltejs/kit').SSRManifest} T */
```

Comment:
I don't think it's correct to use `@template` here. That's for type parameters and not template literals: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template. Unless maybe I'm missing where the `T` is?

Reply:
it's a hack to get VSCode to create a jump link for the SSRManifest. setting it as @type gave an error because the string is not the type SSRManifest